crates/rnote-ui/src/dialogs/export.rs

```diff
@@ -506,7 +512,14 @@ pub(crate) async fn dialog_export_doc_pages_w_prefs(appwindow: &RnAppWindow, can
                                 log::error!("exporting document pages failed, Error: `{e:?}`");
                                 appwindow.overlays().dispatch_toast_error(&gettext("Exporting document pages failed"));
                             } else {
-                                appwindow.overlays().dispatch_toast_text(&gettext("Exported document pages successfully"), crate::overlays::TEXT_TOAST_TIMEOUT_DEFAULT);
+appwindow.overlays().dispatch_toast_w_button_singleton(&gettext("Exported document pages successfully"), &gettext("View in file manager"), clone!(@weak canvas, @weak appwindow => move |_reload_toast| {
+                                    let dir_path = dir.path().unwrap(); // Have to use a let binding because of 'temporary value freed'
+                                    let folder_path = dir_path.to_str().unwrap();
+                                    if let Err(e) = open::that(folder_path) {
+                                        appwindow.overlays().dispatch_toast_error(&gettext("Failed to open the folder in the file manager"));
+                                        log::debug!("opening the folder in the file manager failed: {e:?}",);
```

Comment:
same thing as above with the error message

Reply:
Fixed by newest commit